Library/Homebrew/cask/lib/hbc/artifact/binary.rb

```diff
@@ -9,7 +9,7 @@ def install_phase
 
       def link
         super
-        FileUtils.chmod "+x", source
+        @command.run!("/bin/chmod", args: ["+x", source])
```

Comment:
This should be handled in a similar way as https://github.com/Homebrew/brew/blob/a365e16b48c5ee3461fdda48d8aca58df954c4eb/Library/Homebrew/cask/lib/hbc/artifact/moved.rb#L35-L39.

Also, this should be skipped if it is already executable (`source.executable?`).